packages/xo-web/src/common/xo/index.js

```diff
@@ -359,7 +359,18 @@ export const subscribeRemotes = createSubscription(() => _call('remote.getAll'))
 
 export const subscribeRemotesInfo = createSubscription(() => _call('remote.getAllInfo'))
 
-export const subscribeProxies = createSubscription(() => _call('proxy.getAll'))
+export const subscribeProxies = createSubscription(async () => {
+  const { user } = store.getState()
+  if (user != null && user.permission !== 'admin') {
+    return
+  }
```

Comment:
If `user == null`, we should probably also abort.

Reply:
By using the suggested change by MathieuRA (`isAdmin(store.getState())`), I think this check will also be done (`user && user.permission === 'admin'`) ?